<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusinessDialog</class>
 <widget class="QDialog" name="BusinessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>号码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txi_no">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txi_passwd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5"/>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>办理内容：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txi_comment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
    </widget>
    <widget class="QTextEdit" name="txi_content">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QComboBox" name="cbx_service">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>选择套餐：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>显示套餐：</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>141</width>
       <height>36</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rdo_service_type_on">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>常用</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdo_service_type_off">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>所有</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>证件类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbx_card_type">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>31</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>客户姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txi_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>证件号码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txi_card_no">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>新增亲情号码：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txi_add_no">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>删除亲情号码：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txi_delete_no">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>提交办理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
